chaoslib/run.py

```diff
@@ -331,10 +331,19 @@ def _run(self, strategy: Strategy, schedule: Schedule,  # noqa: C901
         hypo_pool = get_hypothesis_pool()
         continous_hypo_event = threading.Event()
 
-        dry = experiment.get("dry", False)
-        if dry:
+        dry = experiment.get("dry", "no-dry")
```

Comment:
```suggestion
        dry = experiment.get("dry", None)
```

to reflect the switch from `"no-dry"` to `None`